client/app/_services/launcher.service.js

```diff
@@ -46,6 +47,10 @@
             return $http.post(API_URL + '/api/launchers/build' + (providerId ? `?providerId=${providerId}` : ''), launcher).then(UtilService.handleSuccess, UtilService.handleError('Unable to build with launcher'));
         }
 
+        function setFavouriteLauncher(id, params) {
+            return $http.patch(API_URL + '/api/launchers/' + id, params).then(UtilService.handleSuccess, UtilService.handleError('Unable to update launcher parameter'));
```

Comment:
Please use backtick instead of string concatenation

Reply:
fixed